src/BlazorUI/Bit.BlazorUI/Components/Tooltip/BitTooltip.Fluent.scss

```diff
@@ -0,0 +1,239 @@
+            box-shadow: rem(-2px) rem(2px) rem(2px) 0 rgba(0, 0, 0, 0.1);
+        }
+    }
+
+    .tooltip-position-bottom {
+        bottom: rem(-47px);
+        left: 50%;
+        transform: translateX(-50%);
+        box-shadow: 0 rem(-2px) rem(2px) 0 rgba(0, 0, 0, 0.1), 0 rem(1px) rem(1px) 0 rgba(0, 0, 0, 0.1);
+
+        &::after {
+            content: "";
+            position: absolute;
+            top: rem(-1px);
+            left: 50%;
+            border: rem(6px) solid;
+            border-color: white transparent transparent white;
+            transform: rotate(45deg) translateX(-50%);
+            box-shadow: rem(-2px) rem(-2px) rem(2px) 0 rgba(0, 0, 0, 0.1);
+        }
+    }
+
+    .tooltip-position-bottom-left {
+        bottom: rem(-47px);
+        right: 95%;
+        box-shadow: 0 rem(-2px) rem(2px) 0 rgba(0, 0, 0, 0.1), 0 rem(1px) rem(1px) 0 rgba(0, 0, 0, 0.1);
+
+        &::after {
+            content: "";
+            position: absolute;
+            top: rem(-1px);
+            right: rem(-2px);
+            border: rem(6px) solid;
+            border-color: white transparent transparent white;
+            transform: rotate(45deg) translateX(-50%);
+            box-shadow: rem(-2px) rem(-2px) rem(2px) 0 rgba(0, 0, 0, 0.1);
+        }
+    }
+
+    .tooltip-position-left-bottom {
+        bottom: rem(-27px);
+        right: 104%;
+        box-shadow: rem(2px) 0 rem(2px) 0 rgba(0, 0, 0, 0.1), 0 rem(1px) rem(1px) 0 rgba(0, 0, 0, 0.1);
+
+        &::after {
+            content: "";
+            position: absolute;
+            top: rem(-2px);
+            right: 0%;
+            border: rem(6px) solid;
+            border-color: transparent transparent white white;
+            transform: rotate(-135deg) translateX(-50%);
+            box-shadow: rem(-2px) rem(2px) rem(2px) 0 rgba(0, 0, 0, 0.1);
+        }
+    }
+
+    .tooltip-position-left {
+        top: rem(-4px);
+        right: 104%;
+        box-shadow: rem(2px) 0 rem(2px) 0 rgba(0, 0, 0, 0.1), 0 rem(1px) rem(1px) 0 rgba(0, 0, 0, 0.1);
+
+        &::after {
+            content: "";
+            position: absolute;
+            top: 50%;
+            right: 0%;
+            border: rem(6px) solid;
+            border-color: white transparent transparent white;
+            transform: rotate(135deg) translateX(-50%);
+            box-shadow: rem(-2px) rem(-2px) rem(2px) 0 rgba(0, 0, 0, 0.1);
+        }
+    }
+
+    .tooltip-position-left-top {
+        top: rem(-27px);
+        right: 104%;
+        box-shadow: rem(2px) 0 rem(2px) 0 rgba(0, 0, 0, 0.1), 0 rem(1px) rem(1px) 0 rgba(0, 0, 0, 0.1);
+
+        &::after {
+            content: "";
+            position: absolute;
+            bottom: rem(-2px);
+            right: 0%;
+            border: rem(6px) solid;
+            border-color: white transparent transparent white;
+            transform: rotate(135deg) translateX(-50%);
+            box-shadow: rem(-2px) rem(-2px) rem(2px) 0 rgba(0, 0, 0, 0.1);
```

Comment:
the shared css attributes of the `::after` selector should be encapsulated in a single place.